discretize/MeshIO.py

```diff
@@ -44,6 +44,63 @@ def readCellLine(line):
         # Make the mesh
         tensMsh = TensorMesh([h1, h2, h3], x0)
         return tensMsh
+    
+    @classmethod
+    def readUBC_DC2DMesh(TensorMesh,fileName):
+        """
+        Read UBC GIF 2DTensor mesh and generate 2D Tensor mesh in simpeg
+
+        :param string fileName: path to the UBC GIF mesh file
+        :rtype: TensorMesh
+        :return: SimPEG TensorMesh 2D object
+
+        Created on Thu Nov 12 13:14:10 2015
+
+        @author: dominiquef
+
+        """
+
+        fopen = open(fileName, 'r')
+
+        # Read down the file and unpack dx vector
+        def unpackdx(fid, nrows):
+            for ii in range(nrows):
+
+                line = fid.readline()
+                var = np.array(line.split(), dtype=float)
+
+                if ii==0:
+                    x0= var[0]
+                    xvec = np.ones(int(var[2])) * (var[1] - var[0]) / int(var[2])
+                    xend = var[1]
+
+                else:
+                    xvec = np.hstack((xvec, np.ones(int(var[1])) * (var[0] - xend) / int(var[1])))
+                    xend = var[0]
+
+            return x0, xvec
+
+        # Start with dx block
+        # First line specifies the number of rows for x-cells
+        line = fopen.readline()
+        nl = np.array(line.split(), dtype=float)
```

Comment:
This is causing the failure (https://travis-ci.org/simpeg/discretize/jobs/229743764#L6327). This should be `dtype=int` ?? 
![image](https://cloud.githubusercontent.com/assets/6361812/26006116/26565da2-36f0-11e7-8a06-5577f1169591.png)